apps/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -468,6 +488,87 @@ export class WorkspaceInstallManager extends BaseInstallManager {
     });
   }
 
+  protected _createPerProjectManifestAsync(
```

Comment:
Add a code comment explaining what a "per-project manifest" is and why we want one. It's not obvious.